tests/e2e/cli-upgrade/chainsaw-test.yaml

```diff
@@ -73,7 +63,11 @@ spec:
               ./odigos install --namespace odigos-test
         - assert:
             file: ../../common/assert/odigos-installed.yaml
-
+    - name: Assert Tempo Is Up
+      try:
+        - assert:
+            timeout: 5m
+            file: ../../common/assert/tempo-running.yaml
```

Comment:
you can even consider moving this part further down to install the demo app and other steps without blocking for tempo to be ready